src/components/shared/ThemeSwitcher.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Select, useTheme } from "@urkellabs/ucl";
+
+export default function NetworkSwitcher(props) {
+  let [, toggleTheme, current] = useTheme();
```

Comment:
extra comma?

Reply:
3 items get returned from useTheme (themeObject, toggleTheme, currentTheme). In order to get the last 2 without the first this is the syntax unfortunately. Wish it were more like rust where we could do `_`, but apparently this is what JS thinks is right lol.

Reply:
mismatched file name and function name?